tests/foreman/cli/test_bootstrap_script.py

```diff
@@ -15,62 +15,205 @@
+        )
+        rh_repo = entities.Repository(id=rh_repo_id).read()
+
+        # Create essentials for an activation key
+        lce = entities.LifecycleEnvironment(organization=cls.org.id).create()
+        cv = entities.ContentView(
+            organization=cls.org.id,
+            repository=[rh_repo],
+        ).create()
+        cv.publish()
+        cv = cv.read()
+        promote(cv.version[0], environment_id=lce.id)
+        cv = cv.read()
+
+        # Create an activation key and set it to not auto-attach
+        cls.ak = entities.ActivationKey(
+            environment=cls.org.library,
+            auto_attach=False, content_view=cv, organization=cls.org.id,
+            ).create()
+
+        prod = rh_repo.product.read()
+        subs = entities.Subscription().search(
+            query={'search': 'name~{0}'.format(prod.name)}
+        )
+
+        cls.ak.add_subscriptions(data={'subscriptions': [{'id': subs[0].id}]})
+
+        # Search for SmartProxy
+        proxy = entities.SmartProxy().search(
+            query={
+                u'search': u'name={0}'.format(
+                    settings.server.hostname)
+                }
+            )
+
+        # Get the arch ID
+        architecture = entities.Architecture().search(
+            query={
+                u'search': u'name="{0}"'.format(DEFAULT_ARCHITECTURE)
+                   }
+        )[0].read()
+
+        # Create the OS
+        os = entities.OperatingSystem().search(query={
+            u'search': u'name="RedHat" AND (major="{0}")'
+            .format(RHEL_7_MAJOR_VERSION)
+            })[0].read()
+
+        cls.domain = entities.Domain(
+            location=[cls.loc],
+            organization=[cls.org],
+                ).create()
+
+        # Create a host group
+        cls.hostgroup = entities.HostGroup(
+            architecture=architecture,
+            content_source=proxy[0].id,
+            domain=[cls.domain.id],
+            location=[cls.loc],
+            content_view=cv,
+            lifecycle_environment=lce.id,
+            organization=[cls.org],
+            operatingsystem=os,
+                ).create()
+
+    @tier4
     def test_positive_register(self):
-        """System is registered
+        """System is registered.
 
         :id: e34561fd-e0d6-4587-84eb-f86bd131aab1
 
         :Steps:
 
-            1. assure system is not registered
-            2. register a system
+            1. create a container with host name
+            2. register system using bootstrap.py
+            3. assert system has been registered
 
         :expectedresults: system is registered, host is created
 
-        :CaseAutomation: notautomated
+        :CaseAutomation: automated
 
-        :CaseImportance: Critical
+        :CaseImportance: High
         """
+        my_host = Container(agent=True)
+        host_name = my_host.execute("hostname")
+        my_fqdn = host_name.strip() + '.' + self.domain.name + '.' 'com'
```

Comment:
Wouldn't `format` be cleaner?
```python
my_fqdn = "{}.{}.com".format(host_name.strip(), self.domain.name)
```
f-string would be even more readable, but I think it trips betlegeuse and we shouldn't use it in test code for now.

Reply:
Hello @mirekdlugosz 
As I am using the .format method, could you please elaborate?
Thank you

Reply:
I mean this particular line.
You have:
```python
my_fqdn = host_name.strip() + '.' + self.domain.name + '.' 'com'
```
My proposal is:
```python
my_fqdn = "{}.{}.com".format(host_name.strip(), self.domain.name)
```

Reply:
oh yes, now I see it. Will do, thank you